omnigibson/robots/tiago.py

```diff
@@ -550,16 +669,30 @@ def finger_joint_names(self):
 
     @property
     def usd_path(self):
-        return os.path.join(gm.ASSET_PATH, "models/tiago/tiago_dual_omnidirectional_stanford/tiago_dual_omnidirectional_stanford_33.usd")
+        return os.path.join(gm.ASSET_PATH, "models/tiago/tiago_dual_omnidirectional_stanford/tiago_dual_omnidirectional_stanford_33_with_wrist_cam.usd")
```

Comment:
perhaps we should keep the "vanilla" model? unless we had a eye-in-hand also to Fetch

Reply:
I added a variant option to solve this.